react/wbooks/src/app/components/Routes/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+
+import { Routes } from '../../../constants';
+import SignUp from '../SignUp';
+
+function AppRoutes() {
+  return (
+    <Router>
+      <Switch>
+        <Route path={Routes.LOGIN} exact component={() => <p>Not implemented yet :D</p>} />
```

Comment:
im ok with this :v but its better to use `render=` instead of `component=` if you are using this arrow function instant component ✌️

Reply:
why is better?